src/engine/Server.cpp

```diff
@@ -4,21 +4,11 @@
 
 #include "./Server.h"
 
-#include <re2/re2.h>
-
-#include <algorithm>
 #include <cstring>
 #include <sstream>
 #include <string>
-#include <thread>
 #include <vector>
 
-#include "../parser/ParseException.h"
-#include "../util/BoostHelpers/AsyncWaitForFuture.h"
-#include "../util/HttpServer/UrlParser.h"
-#include "../util/Log.h"
-#include "../util/StringUtils.h"
-#include "../util/json.h"
```

Comment:
Do you have a tool that tracks unused includes?
Which one is it?

Reply:
CLion greys out any unused headers. Not sure if this is 100% reliable all the time, but it worked without problems so far